superbench/executor/executor.py

```diff
@@ -200,6 +200,7 @@ def exec(self):
             benchmark_config = self._sb_benchmarks[benchmark_name]
             benchmark_results = list()
             self.__create_benchmark_dir(benchmark_name)
+            os.chdir(self.__get_benchmark_dir(benchmark_name))
```

Comment:
may need to change back in the end of loop to pass tests..

Reply:
Sure